packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -167,6 +169,9 @@ export class DeviceSession implements Disposable {
     devtools.onEvent("RNIDE_isProfilingReact", (isProfiling) => {
       this.deviceSessionDelegate.onAppEvent("isProfilingReact", isProfiling);
     });
+    devtools.onEvent("RNIDE_routeListRetrieved", (payload) => {
```

Comment:
I'd collocate this with `RNIDE_navigationChanged` event from above because they are related. We could also rename this such that it starts with "navigation", for example `RNIDE_navigationRouteListUpdated`